proposals/nnnn-Make-RPC-Setters-Chainable.md

```diff
@@ -0,0 +1,69 @@
+# Make RPC Setters Chainable
+
+* Proposal: [SDL-NNNN](nnnn-Make-RPC-Setters-Chainable.md)
+* Author: [Bilal Alsharifi](https://github.com/bilal-alsharifi)
+* Status: **Awaiting review**
+* Impacted Platforms: [Java Suite]
+
+## Introduction
+
+This proposal is to enhance how the params' setters for the RPC classes work to make creating new RPCs easier. This can be done by making each setter return an instance of the RPC object. By doing that, creating an RPC and setting its params can all be done in a one-liner. This reduces the amount of boilerplate code that developers need to write to create RPCs.
```

Comment:
```suggestion
This proposal is to enhance how the params' setters for the RPC classes work to make creating new RPCs easier. This can be done by making each setter return an instance of the RPC object. By doing that, creating an RPC and setting its params can all be done in one line of code. This reduces the amount of boilerplate code that developers need to write to create RPCs.
```